src/commons/assessmentWorkspace/AssessmentWorkspace.tsx

```diff
@@ -757,6 +757,9 @@ const AssessmentWorkspace: React.FC<AssessmentWorkspaceProps> = props => {
       ? {
           editorSessionId: '',
           editorValue: props.editorValue!,
+          sourceChapter: question.library.chapter || 4,
```

Comment:
Hmm, the documentation says that DEPLOYMENT element is used for getting the "chapter"; not sure if this is right.

Reply:
If believe the XML parses the Source Chapter from the DEPLOYMENT tag, and ultimately stores it as a jsonb (with the rest of the library details)

During assessment retrieval, this is how the chapter (and rest of the library details) per question are sent to the frontend

can @YaleChen299 double confirm?

Reply:
Yes. `interpreter` attribute under `DEPLOYMENT` is parsed as `library.chapter`.